.github/workflows/test-integrations.yml

```diff
@@ -387,6 +388,8 @@ jobs:
       fail-fast: false
       matrix:
         test-cases: ${{ fromJSON(needs.generate-compatibility-job-matrices.outputs.compatibility-matrix) }}
+    env:
+      ENVOY_VERSION: "1.24.6"
```

Comment:
IICR, 1.15 is currently compatible w/ 1.25, 1.24, and 1.23. I would vote to have this track the most recent supported version.

Reply:
Thanks, @DanStough . Since the upgrade test tests from 1.14 and 1.15 to main, I will use the most recent version 1.14 supports, which is 1.25.